packages/block-library/src/gallery/transforms.js

```diff
@@ -33,6 +34,33 @@ const parseShortcodeIds = ( ids ) => {
 	return ids.split( ',' ).map( ( id ) => parseInt( id, 10 ) );
 };
 
+function transformV1FormatFromThirdPartyBlocks( block ) {
```

Comment:
Would it help future archaeologists to have a JS Doc or comment describing the reason for this, or is the function name enough?

Happy to leave as the latter if you think it's obvious.

Reply:
My future self may work out the what from the method name, but probably not the why - have added a comment.